CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ Release report: TBD
 
 - Add Windows location wildcards tests ([#4263](https://github.com/wazuh/wazuh-qa/pull/4263)) \- (Tests + Framework)
 - New 'SCA' test suite and framework. ([#3566](https://github.com/wazuh/wazuh-qa/pull/3566)) \- (Framework + Tests)
+- Add tests for merged.mg file generation. ([#3278](https://github.com/wazuh/wazuh-qa/pull/4129)) \- (Tests)
```

Comment:
```suggestion
- Add tests for merged.mg file generation. ([#3278](https://github.com/wazuh/wazuh-qa/pull/4129)) \- (Tests)
- Add Windows location wildcards tests ([#4263](https://github.com/wazuh/wazuh-qa/pull/4263)) \- (Tests + Framework)
- New 'SCA' test suite and framework. ([#3566](https://github.com/wazuh/wazuh-qa/pull/3566)) \- (Framework + Tests)
```

Changes should go upwards

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/4129/commits/f19249d8b111135b9e0f42fcc3dba0c6fbbd7c51